### PR TITLE
refactor(components): popover showcase refactor

### PR DESCRIPTION
**Pull request by 32penkin**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/kittenTricks/blob/master/CONTRIBUTING.md) guide.

### CODE REVIEW

File: src/containers/components/popover/showcase/popoverShowcase.component.tsx [hunk 1]
```diff
@@ -76,4 +84,7 @@ const styles = StyleSheet.create({
   content: {
     width: 228,
   },
+  button: {
+    width: 106,
```
Comment (artyorsh) [inline src/containers/components/popover/showcase/popoverShowcase.component.tsx:88]: why?

File: src/core/navigation/routes.tsx [hunk 2]
```diff
@@ -467,7 +467,8 @@ const MenuNavigator: ReactNavigationContainer = createBottomTabNavigator({
 
 const AppNavigator: ReactNavigationContainer = createStackNavigator({
   ['Home']: {
-    screen: ListContainer,
+    screen: MenuNavigator,
+    // screen: PopoverContainer,
```
Comment (artyorsh) [inline src/core/navigation/routes.tsx]: undo

**Review by artyorsh — APPROVED** [review]

File: src/core/navigation/routes.tsx (hunk 2 above)
Comment (32penkin) [reply src/core/navigation/routes.tsx]: Done

File: src/containers/components/popover/showcase/popoverShowcase.component.tsx (hunk 1 above)
Comment (32penkin) [reply src/containers/components/popover/showcase/popoverShowcase.component.tsx:88]: Just to fix Button width :)

**Pull request merged**